packages/envied_generator/lib/src/generate_field_encrypted.dart

```diff
@@ -124,7 +173,13 @@ Iterable<Field> generateFieldsEncrypted(FieldElement field, String value) {
         (FieldBuilder fieldBuilder) => fieldBuilder
           ..static = true
           ..modifier = FieldModifier.final$
-          ..type = refer(field.type is DynamicType ? '' : 'String')
+          ..type = field.type is DynamicType
+              ? null
```

Comment:
Not sure this is correct. Should probably be `refer('null')` or something. Check `code_builder` for examples.

Reply:
This is basically not assigning a type for the `fieldBuilder`. I tried `refer('null') but it does not work.

`refer('')` does work, but that seems a little weird.

I don't understand why we have support for dynamic for obfuscated Strings, seems a little weird.

Reply:
I'm not finding any examples for not putting a type on a field builder in their repo

Reply:
> I don't understand why we have support for dynamic for obfuscated Strings, seems a little weird.

That's a question for @ThexXTURBOXx who wrote the original obfuscator. All I did was just convert it to use `code_builder`.